docs/components/tracked-controls.md

```diff
@@ -0,0 +1,43 @@
+---
+title: tracked-controls
+type: components
+layout: docs
+parent_section: components
+---
+
+The tracked-controls component interfaces with tracked controllers. 
+Uses the Gamepad API to handle tracked controllers. 
+Selects the appropriate controller, applies pose to the entity, observes buttons state and emits appropriate events.
+
```

Comment:
Can we also back-link to Vive and Hand control docs mentioning that they abstract this component